app/validators/answers/option_answer_validator.py

```diff
@@ -10,24 +11,57 @@ class OptionAnswerValidator(AnswerValidator):
     LIST_NAME_MISSING = "List name defined in action params does not exist"
     BLOCK_ID_MISSING = "Block id defined in action params does not exist"
     ANSWER_DEFAULT_MISSING = "Couldn't find matching value for answer default"
+    INVALID_NUMBER_OF_ANSWER_OPTIONS = (
+        "Not enough options. "
+        "{answer_type} requires at least {required_num_options} answer option(s) but got {actual_num_options}"
+    )
+    OPTIONS_DEFINED_BUT_EMPTY = "Answer options defined, but empty"
+    DYNAMIC_OPTIONS_REFERENCES_NON_CHECKBOX_ANSWER = (
+        "Dynamic options references non Checkbox answer"
+    )
+
+    MIN_OPTIONS_BY_ANSWER_TYPE = {"Checkbox": 1, "Radio": 2, "Dropdown": 2}
```

Comment:
With no of errors increasing , i personally prefer to have these error messages have its own class or file. we should really be looking at [Enum](https://docs.python.org/3.11/howto/enum.html)

`MIN_OPTIONS_BY_ANSWER_TYPE` is an odd one out among errors

Reply:
I agree but kept it consistent with what we have. I think it all the validators should be refactored in that way. 

I have made `MIN_OPTIONS_BY_ANSWER_TYPE` a module level constant.